src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/DruidCombatAI.java

```diff
@@ -0,0 +1,340 @@
+        return pathResult;
+    }
+
+    /**
+     * Get combat speed
+     *
+     * @return movent speed
+     */
+    protected double getCombatMovementSpeed()
+    {
+        double levelAdjustment = user.getCitizenData().getCitizenSkillHandler().getLevel(Skill.Mana) * SPEED_LEVEL_BONUS;
+        levelAdjustment += (user.getCitizenData().getWorkBuilding().getBuildingLevel() * 2 - 1) * SPEED_LEVEL_BONUS;
+
+        levelAdjustment = Math.min(levelAdjustment, 0.3);
+        return COMBAT_SPEED + levelAdjustment;
+    }
+
+    @Override
+    protected boolean isAttackableTarget(final LivingEntity entity)
+    {
+        return (AbstractEntityAIGuard.isAttackableTarget(user, entity) || (entity instanceof IThreatTableEntity && ((IThreatTableEntity) entity).getThreatTable().getTarget() != null ) || entity instanceof PlayerEntity) && !wasAffectedByDruid(entity);
+    }
+
+    @Override
+    protected boolean searchNearbyTarget()
+    {
+        if (checkForTarget())
+        {
+            return true;
+        }
+
+        final List<LivingEntity> entities = user.level.getLoadedEntitiesOfClass(LivingEntity.class, getSearchArea());
+
+        if (entities.isEmpty())
+        {
+            return false;
+        }
+
+        int targetsUnderEffect = 0;
+        boolean foundTarget = false;
+        for (final LivingEntity entity : entities)
+        {
+            if (!entity.isAlive())
+            {
+                continue;
+            }
+
+            if (skipSearch(entity))
+            {
+                return false;
+            }
+
+            if (isEntityValidTarget(entity))
+            {
+                if (user.canSee(entity))
+                {
+                    user.getThreatTable().addThreat(entity, 0);
+                    foundTarget = true;
+                }
+            }
+            else if (wasAffectedByDruid(entity))
+            {
+                targetsUnderEffect++;
+            }
+        }
+
+        return foundTarget && targetsUnderEffect <= parentAI.getOwnBuilding().getBuildingLevel() * 2;
+    }
+
+    /**
+     * Check if an entity has one of the potion effects the druid hands out.
+     * @param entity the entity to check for.
+     * @return true if so.
+     */
+    private boolean wasAffectedByDruid(final LivingEntity entity)
+    {
+        return entity.hasEffect(Effects.MOVEMENT_SLOWDOWN) || entity.hasEffect(Effects.SATURATION) || entity.hasEffect(Effects.DAMAGE_BOOST) || entity.hasEffect(Effects.WEAKNESS) || entity.hasEffect(Effects.DAMAGE_RESISTANCE) || entity.hasEffect(Effects.HEAL);
+    }
+
+    @Override
+    protected boolean isWithinPersecutionDistance(final LivingEntity target)
+    {
+        return parentAI.isWithinPersecutionDistance(target.blockPosition(), getAttackDistance());
+    }
+
+    @Override
+    protected boolean skipSearch(final LivingEntity entity)
+    {
+        // Found a sleeping guard nearby
+        if (entity instanceof EntityCitizen)
```

Comment:
might be nice to have druids be the nice guys who are like "let him sleep, he seems tired"

Reply:
made them "forgiving" (small chance to wake them up)